<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvictionWindow</class>
 <widget class="QWidget" name="EvictionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выселение клиента</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="searchType">
       <item>
        <property name="text">
         <string>Номеру комнаты</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Имени</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Фамилии</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Паспорту</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Поиск по</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="searchEdit"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="searchBtn">
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="showAllBtn">
       <property name="text">
        <string>Показать всё</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Введите ID клиента для удаления</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deleteEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBtn">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
